src/bindings/kotlin/templates/macros.kt

```diff
@@ -0,0 +1,45 @@
+{# 
+// Template to call into rust. Used in several places.
+// Variable names in `arg_list_decl` should match up with arg lists
+// passed to rust via `_arg_list_rs_call` (we use  `var_name_kt` in `lower_kt`)
+#}
+
+{%- macro to_rs_call(func) -%}
+_UniFFILib.INSTANCE.{{ func.ffi_func().name() }}({% call _arg_list_rs_call(func.arguments()) -%})
+{%- endmacro -%}
+
+{%- macro to_rs_call_with_prefix(prefix, func) -%}
+_UniFFILib.INSTANCE.{{ func.ffi_func().name() }}(
+    {{- prefix }}{% if func.arguments().len() > 0 %}, {% call _arg_list_rs_call(func.arguments()) -%}{% endif -%}
+)
+{%- endmacro -%}
+
+{%- macro _arg_list_rs_call(args) %}
+    {%- for arg in args %}
+        {{- arg.name()|lower_kt(arg.type_()) }}
+        {%- if !loop.last %}, {% endif %}
+    {%- endfor %}
+{%- endmacro -%}
```

Comment:
This is only called within this file.

This mangles the name in a call to the JNA library in methods/functions/constructors.

```kotlin
lower(initialPoint)
```